personalization-service/src/test/java/de/muenchen/dbs/personalization/checklist/ChecklistServiceTest.java

```diff
@@ -2,28 +2,25 @@
 
 import static de.muenchen.dbs.personalization.checklist.ChecklistTestHelper.createTestChecklist;
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.argThat;
-import static org.mockito.Mockito.doNothing;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.*;
 
 import de.muenchen.dbs.personalization.checklist.domain.Checklist;
+import de.muenchen.dbs.personalization.checklist.domain.ChecklistMapper;
+import de.muenchen.dbs.personalization.checklist.domain.ChecklistMapperImpl;
 import de.muenchen.dbs.personalization.checklist.domain.ChecklistServiceNavigatorReadDTO;
 import de.muenchen.dbs.personalization.common.NotFoundException;
 import de.muenchen.dbs.personalization.servicenavigator.ServiceNavigatorService;
 import java.time.Instant;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Please no asterisk imports

Reply:
done [2ba6a43](https://github.com/it-at-m/dbs/pull/238/commits/2ba6a4368e2189c9f15dd8e6ff49e76513c1f2df)